lib/webclient/features_webclient.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:bytebank/webclient/webclient_config.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:bytebank/models/feature.dart';
+
+final String urlBase = "$urlBaseApi/features";
+
+const _featuresNaoEncontradas = 'Não foram encontradas as features';
+
+//TODO usei o http considerando a sugestão da documentação, tem outra lib recomendada?
+class FeatureWebClient {
+
+  Future<List<Feature>> todas() async {
+    final resposta = await http.get(urlBase);
+    //TODO essa abordagem de verificar o que a resposta indica é normal? Não teria algo como `isSuccessful`?
```

Comment:
If `isSuccessful` depends only on your business logic. 
`http` just makes requests, you have to check if it is a valid response or not.

Reply:
For these project, only the status 200 is enough to represents the success response, so I'm thinking extract the below function in `webclient_config`:

```
bool sucesso(http.Response resposta) => resposta.statusCode == 200;
```

This way, I can reuse at the all web clients, what do you think?

Reply:
Yes, isolating business logic is better in this case.